src/ws-handler.ts

```diff
@@ -589,4 +593,23 @@ export class WsHandler {
 
         return randomNumber(min, max) + '.' + randomNumber(min, max);
     }
+
+    /**
+     * Clear WebSocket timeout.
+     */
+    protected clearTimeout(ws: WebSocket) {
+        if(ws.timeout) {
+            clearTimeout(ws.timeout)
+        }
+    }
+
+    /**
+     * Update WebSocket timeout.
+     */
+    protected updateTimeout(ws: WebSocket) {
+        this.clearTimeout(ws);
+        ws.timeout = setTimeout(() => {
+            ws.close()
+        }, 120_000)
```

Comment:
Nothing to note here. I like the `_` usage here. I think I'll adopt this for my future code instead of `x * 1000`.